### ISSUE TITLE
Create New Employee (Admin Mode) - Toast notification should have role="status"

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#264]
<img width="842" alt="Screen Shot 2020-04-27 at 12 29 22 PM" src="https://user-images.githubusercontent.com/31868510/80505192-01d7fb00-8942-11ea-8487-c2573c6a468f.png">

Currently the toast has a role of "alert", which will interrupt users using screen readers. Notifications, validation messages and status' conveyed through toasts should always use role="status" to ensure that the information is politely conveyed to the user after they have completed their actions.

You can read more about a11y and toasts here:

https://www.scottohara.me/blog/2019/07/08/a-toast-to-a11y-toasts.html
